contracts/GVault.sol

```diff
@@ -77,7 +77,7 @@ contract GVault is Constants, ERC4626, StrategyQueue, Ownable, ReentrancyGuard {
         uint256 debtPaid,
         uint256 debtAdded,
         uint256 lockedProfit,
-        uint256 excessLoss
+        uint256 lockedProfitBeforeLoss
```

Comment:
After reviewing the full code, indeed I would need the excessLoss only in case the `_loss` > `lockedProfitBeforeLoss` in order to handle a negative lockedProfit.

I think the code should be something like this:

```
        uint256 lockedProfitBeforeLoss = _calculateLockedProfit() +
            _calcFees(_gain);
        uint256 excessLoss = 0;
        if (lockedProfitBeforeLoss > _loss) {
            lockedProfit = lockedProfitBeforeLoss - _loss;
        } else {
            lockedProfit = 0;
            excessLoss = _loss - lockedProfitBeforeLoss;
        }

        emit LogStrategyHarvestReport(
            msg.sender,
            _gain,
            _loss,
            debtPayment,
            credit,
            lockedProfit,
            excessLoss
        );
```

With this code change, I just need to do `lockedProfit` - `excessLoss` to show the final profit.

This would be the expected outcome for each case:

Case 1: Gain
- Before harvest, we had lockedProfit of 10
- After harvest, we have a gain of 5
`lockedProfit` = 15
`excessLoss` = 0

Case 2: Loss < lockedProfitBeforeLoss
- Before harvest, we had lockedProfit of 10
- After harvest, we have a loss of 7
`lockedProfit` = 3
`excessLoss ` = 0

Case 3:  Loss > lockedProfitBeforeLoss
- Before harvest, we had lockedProfit of 10
- After harvest, we have a loss of 14
`lockedProfit` = 0
`excessLoss ` = 4